tygo/package_generator.go

```diff
@@ -26,15 +26,25 @@ func (g *PackageGenerator) Generate() (string, error) {
 
 			// GenDecl can be an import, type, var, or const expression
 			case *ast.GenDecl:
-				if x.Tok == token.VAR || x.Tok == token.IMPORT {
+				if x.Tok == token.IMPORT {
 					return false
 				}
+				isEmit := false
+				if x.Tok == token.VAR {
```

Comment:
Perhaps this makes sense to support `token.CONST` too? (or perhaps there is a good reason not to?)

Reply:
Consts are already handled to export the const definition itself, so I'm inclined to not handle them here as it makes it more complex it we have to decide which way to handle it.